viper.py

```diff
@@ -724,6 +750,38 @@ def fit_chunk(o, chunk, obsname, targ=None, tpltarg=None):
     print(file=parunit)
     #vpr.plot_rvo(rv, e_rv)
 
+
+if createtpl:
+    wtpl_new = {}
+    tpl_new = {}
+    for o in orders:
+        gplot.reset()
+        gplot.key("title 'order: %s' noenhance" % (o))
+        wt = np.array(list(tpl_all[o,0].values())) 	# wavelength
+        ft = np.array(list(tpl_all[o,1].values()))	# data
+        mt = np.array(list(tpl_all[o,2].values()))	# weighting
+        mt[np.isnan(ft)] = np.nan
+        mt[ft<0] = np.nan
+
+        mt[ft>1.15] = np.nanmin(mt)/10.
```

Comment:
I don't full understand this. Is this a downweighting for emission feature? Assuming 1 is continuum? Comment this line.

Reply:
Unfortunately, as the telluric correction is not perfect and problematic especially for strong tellurics, the corrected spectrum sometimes look like in the plot below (green line). Here the wrong telluric model causes outlier, which I down weigth here.

![grafik](https://user-images.githubusercontent.com/77435060/201124355-cd05ef76-23f9-4a11-8b8c-19563ec9147e.png)

Reply:
Ok, for now.
But I will suggest to downweight with the telluric spectrum itself, instead of absolute flux values.
To be continued in #26.

Note `np.nanmin(mt)/10.` will be 0, when following my previous suggestion to use `0` instead `nan` weights.

Reply:
Actually, I downweight with the telluric spectrum. But more about that in https://github.com/mzechmeister/viper/issues/26.

I know. That's why I said  I would need more testing to see, if that causes any problems.